shivi_khanuja/car.py

```diff
@@ -0,0 +1,52 @@
+class Car(object):
+    def __init__(self, price, speed, fuel, mileage):
+        self.price = price
+        self.speed = speed
+        self.fuel = fuel
+        self.mileage = mileage
+        if self.price > 10000 :
+            self.tax = 0.15
+        else:
+            self.tax = 0.12            
+
+
+    def display(self):
+        print "Price is: ${} ".format(self.price + self.price * self.tax)
+        print 'Max Speed is: ' + str(self.speed) + 'mph'
+        print 'Mileage is: ' + str(self.mileage) + 'miles'
+        return self
+
+    def gas(self):
+      if self.fuel == 10:
+         print "Fuel is: {} gal ".format(self.fuel) +  "Kind of Full"
+
+      elif self.fuel == 15:
+         print "Fuel is: {} gal ".format(self.fuel) +  "Full"
+
+      elif self.fuel == 5:
+         print "Fuel is: {} gal ".format(self.fuel) +  "Not Full"
+      else:       
+        print "Fuel is: {} gal ".format(self.fuel) +  "Empty"
+        return self

```

Comment:
When using `.format()` you don't need to concatenate on the end.  This line
``` python
print "Fuel is: {} gal".format(self.fuel) + "Not Full"
```
could have been
``` python
print "Fuel is: {} gal Not Full".format(self.fuel)
```